### PR TITLE
Add STF information on 'Fingertips' page

### PR DESCRIPTION
**Pull request by rnzenha-s**
## Proposed changes

Add a section about STF sensors on the fingertips page.
 
## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [x] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [ ] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [x] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

File: docs/user_guide/sd_fingertips.rst [hunk 1]
```diff
@@ -1,6 +1,99 @@
...
+STF Sensor
+----------
+The Shadow Tactile Fingertip (STF) sensors are magnetic based tactile sensors that are fitted
+in the standard fingertips or (slighlty bigger) thumb shapes. 
+
+.. figure:: ../img/sd_stf_sensors.png
+    :width: 40%
+
+It leverages 17 magnetic sensors evenly distributed at its core. For each sensor, a magnet is
+placed on top of a layer of silicone (flesh) that is then recovered by another layer of
```
Comment (niko-holmes) [inline docs/user_guide/sd_fingertips.rst]: I would suggest "covered" instead of "recovered"

**Review by niko-holmes — COMMENTED** [review]
Very nice looking figures

File: docs/user_guide/sd_fingertips.rst [hunk 2]
```diff
@@ -1,6 +1,100 @@
...
+STF Sensor
+----------
+The Shadow Tactile Fingertip (STF) sensors are magnetic-based tactile sensors fitted in the
+standard fingertips or (slightly bigger) thumb shapes.
+
+
+.. figure:: ../img/sd_stf_sensors.png
+    :width: 40%
+
+It leverages 17 magnetic sensors evenly distributed at its core. For each sensor, a magnet
```
Comment (dg-shadow) [inline docs/user_guide/sd_fingertips.rst]: The word "leverages" feels very clumsy. "uses"?  "comprises"? "is made from"?

File: docs/user_guide/sd_fingertips.rst (hunk 1 above)
Comment (rnzenha-s) [reply docs/user_guide/sd_fingertips.rst]: Thanks! Addressed in https://github.com/shadow-robot/sr_dexterous_hand_documentation/pull/179/commits/66bb6dfef57d154b72092a475a02c1296d76c375

File: docs/user_guide/sd_fingertips.rst (hunk 2 above)
Comment (rnzenha-s) [reply docs/user_guide/sd_fingertips.rst]: Thanks! Addressed in https://github.com/shadow-robot/sr_dexterous_hand_documentation/commit/66bb6dfef57d154b72092a475a02c1296d76c375

**Review by niko-holmes — APPROVED** [review]

File: docs/user_guide/sd_fingertips.rst [hunk 3]
```diff
@@ -1,9 +1,115 @@
...
+   A few caveats should be considered when operating these sensors:
+
+   • When powered up, the electronic components of the sensors warm up until they reach a stable temperature. During this period, the sensors' magnetic readings are prone to changes.
+   • Upon powering up the hand, allow 30 minutes for the sensors' internal temperatures to stabilise.
+   • Changes to the room temperature and between the room temperature and the temperature of the interacted objects may also incur changes in the magnetic readings until the internal temperature of the sensors stabilises.
+   • The sensors are protected against cross-interference with each other. However, the sensors' magnetic readings are prone to interference from strong external magnetic fields, including interference from the magnetic sensors installed in Shadow hands' joints.
+
+The Shadow Tactile Fingertip (STF) sensors are magnetic-based tactile sensors fitted in the
+standard fingertips or (slightly bigger) thumb shapes.
+
```
Comment (dg-shadow) [inline docs/user_guide/sd_fingertips.rst:17]: Not sure about this but should we mention the dropped frames that result from these sensors??

Comment (rnzenha-s) [reply docs/user_guide/sd_fingertips.rst:17]: Ah yes, good point. We probably should

**Review by dg-shadow — APPROVED** [review]

**Pull request merged**